src/client/index.css

```diff
@@ -2,7 +2,7 @@
 @import 'theme.css';
 
 * {
-  font-family: 'Montserrat', sans-serif;
+  /* font-family: 'Montserrat', sans-serif; */
```

Comment:
Also remember to remove all your unused code

Reply:
ya i removed it.